src/Tests/Microsoft.NET.Publish.Tests/PublishItemsOutputGroupTests.cs

```diff
@@ -144,7 +144,47 @@ public void GroupPopulatedCorrectlyWithSingleFile()
             }
         }
 
-        private TestProject SetupProject()
+        [CoreMSBuildAndWindowsOnlyFact]
+        public void GroupBuildsWithoutPublish()
+        {
+            var testProject = this.SetupProject(addCopyFilesTargets: false);
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            var restoreCommand = new RestoreCommand(Log, testAsset.Path, testProject.Name);
+            restoreCommand
+                .Execute()
+                .Should()
+                .Pass();
+
+            var buildCommand = new BuildCommand(Log, testAsset.Path, testProject.Name);
```

Comment:
I think this test should be parametrized so that it runs in both single-file and non-single-file scenarios.

Reply:
The SingleFile property in the first version was actually irrelevant to what this test is verifying.  I removed it.